src/Microsoft.Identity.Abstractions/CompatibilitySuppressions.xml

```diff
@@ -1,6 +1,30 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
```

Comment:
Are these still necessary?

Reply:
Oh, I think they are because you aren't putting the Patch APIs in the net8 target.

We should fix the .tt file to also include those for net8.

Reply:
Thanks for catching that - I've updated the tt file

Reply:
The remaining suppresions are the new JsonTypeInfo interfaces